src/extension.ts

```diff
@@ -110,4 +114,91 @@ export const activate = () => {
         if (!getExtensionSetting('runOnSave') || reason === vscode.TextDocumentSaveReason.AfterDelay) return
         waitUntil(performFixes())
     })
+
+    vscode.workspace.onDidChangeTextDocument(
+        ({ contentChanges, document }) => {
```

Comment:
Also destructure `reason` here, and do check like in https://github.com/zardoy/vscode-better-snippets/blob/829a6bde0886a007e1cd7daf9a59ce850e601ea6/src/extension.ts#L314